README.md

```diff
@@ -20,23 +20,28 @@ Current contributors include:
 
 The interface is proposed to contain the following features in its first iteration:
 
-- multiple interactive scatter plots with selectable points
-- colouring of samples by metadata or expression values
-- dynamic updating of cell-level metadata on hover/click
-- zooming in to a particular subregion of the plot, if requested
+:white_check_mark: multiple interactive scatter plots with selectable points
 
-Scatter plots can be generated from reduced dimensionality data, or with biaxial plots of existing metadata columns.
+:white_check_mark: colouring of samples by metadata or expression values
+
+dynamic updating of cell-level metadata on hover/click
+
+:white_check_mark: zooming in to a particular subregion of the plot, if requested
+
+:white_check_mark: scatter plots can be generated from reduced dimensionality data, or with biaxial plots of existing metadata columns.
 
 ## Gene-based visualization
 
 The interface is proposed to contain the following features in its first iteration:
 
-- boxplots of expression values for a single gene, stratified by metadata level
-- heatmaps of multiple genes for groups of cells or for individual ordered cells 
-- integrated brushing in cell-based scatter plots with cell identities in gene-level plots
+:white_check_mark: boxplots of expression values for a single gene, stratified by metadata level
+
+heatmaps of multiple genes for groups of cells or for individual ordered cells 
```

Comment:
Not sure how this fits into our framework, TBH.

Reply:
at some point, it could be nice to bring in some pathway annotations (GO, KEGG, etc.).
Although that opens all kinds of issues w.r.t. species, identifiers, just to mention a couple.

Reply:
Well, you'll need some genes in order to do gene set enrichment analyses, and to get that you'll need DE analyses, e.g., between clusters. That's too deep into analysis.